subsystems/sidecar/index.js

```diff
@@ -892,6 +894,20 @@ class Sidecar extends ReadyResource {
     yield * preferences.entries()
   }
 
+  #setKnownNodes (dht) {
+    if (dht) knownNodes.set('dht', dht)
+  }
+
+  #getKnownNodes () {
+    let dht = this.swarm.dht.toArray({ limit: KNOWN_NODES_LIMIT })
+    if (dht) {
+      this.#setKnownNodes(dht)
```

Comment:
Dont think it should set them here, just on teardown explicitly

Reply:
so function isn't needed then, just do this all inline in shutdown